### PR TITLE
Update machine specs to make them more clear for linux build machines

### PR DESCRIPTION
**Pull request by craastad**
Main changes are in these two screenshots:
Screen Shot 2021-03-16 at 1 27 00 PM

Screen Shot 2021-03-16 at 1 27 18 PM

### CODE REVIEW

File: content/specs/machine-type.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Configuring build machine type
+
+In `codemagic.yaml`, the build machine type can be specified with [Instance type](../getting-started/yaml#instance-type).
+
+For Flutter projects configured via the Flutter workflow editor, the build machine type can be selected in **App settings > Workflow settings > Machine**.
+
+## Mac Mini and Mac Pro
+
+Codemagic offers two types of macOS machines for running builds: Mac Mini (macOS standard VM, default) and Mac Pro (macOS premium VM). Specifications for these machines are available [here](../specs/versions/#hardware).
```
Comment (mikhail-tokarev) [inline content/specs/machine-type.md:17]: `Mac mini`

File: content/specs/machine-type.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Codemagic offers two types of macOS machines for running builds: Mac Mini (macOS standard VM, default) and Mac Pro (macOS premium VM). Specifications for these machines are available [here](../specs/versions/#hardware). 
+
+{{<notebox>}}
+Mac Pro machines are only available for teams and users that have [billing enabled](../billing/billing). See the [pricing page](https://codemagic.io/pricing/) for the per minute rate.
+{{</notebox>}}
+
+## Linux Standard and Linux Premium
+
+Codemagic offers two types of linux machines for running builds: Linux standard VM and Linux premium VM. Linux machines support nested virtualisation, including Android emulator support with hardware acceleration. Linux machines do not support macOS specific software, such as brew and xcode, and cannot be used to build iOS artifacts. Specifications for these machines are available [here](../specs/versions-linux/#hardware).
```
Comment (mikhail-tokarev) [inline content/specs/machine-type.md:25]: Xcode

File: content/specs/versions-linux.md [hunk 3]
```diff
@@ -3,13 +3,13 @@ description: A list of tools available out-of-the-box on Codemagic build machine
 title: Linux (beta) build machine specification
```
Comment (mikhail-tokarev) [inline content/specs/versions-linux.md:3]: let's remove beta

**Review by mikhail-tokarev — APPROVED** [review]

File: content/specs/machine-type.md (hunk 1 above)
Comment (craastad) [reply content/specs/machine-type.md:17]: Why is it `Mac mini` and `Mac Pro`? Should it also be `Mac pro`?

File: content/specs/versions-linux.md (hunk 3 above)
Comment (craastad) [reply content/specs/versions-linux.md:3]: Good catch :+1:

File: content/specs/machine-type.md (hunk 1 above)
Comment (mikhail-tokarev) [reply content/specs/machine-type.md]: trademark I guess: `Mac mini` but `Mac Pro`

File: content/specs/machine-type.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Codemagic offers two types of macOS machines for running builds: Mac mini (macOS standard VM, default) and Mac Pro (macOS premium VM). Specifications for these machines are available [here](../specs/versions/#hardware). 
+
+{{<notebox>}}
+Mac Pro machines are only available for teams and users that have [billing enabled](../billing/billing). See the [pricing page](https://codemagic.io/pricing/) for the per minute rate.
+{{</notebox>}}
+
+## Linux Standard and Linux Premium
+
+Codemagic offers two types of linux machines for running builds: Linux standard VM and Linux premium VM. Linux machines support nested virtualisation, including Android emulator support with hardware acceleration. Linux machines do not support macOS specific software, such as brew and Xcode, and cannot be used to build iOS artifacts. Specifications for these machines are available [here](../specs/versions-linux/#hardware).
```
Comment (helinanever) [inline content/specs/machine-type.md:25]: Codemagic offers two types of linux machines... -> Linux

**Review by helinanever — COMMENTED** [review]
Two minor comments, but other than that looks good to me.

File: content/specs/machine-type.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+## Configuring build machine type
+
+In `codemagic.yaml`, the build machine type can be specified with [Instance type](../getting-started/yaml#instance-type).
+
+For Flutter projects configured via the Flutter workflow editor, the build machine type can be selected in **App settings > Workflow settings > Machine**.
+
+## Mac Mini and Mac Pro
+
+Codemagic offers two types of macOS machines for running builds: Mac mini (macOS standard VM, default) and Mac Pro (macOS premium VM). Specifications for these machines are available [here](../specs/versions/#hardware). 
+
```
[the remark this answers is not in the document]
Comment (craastad) [reply content/specs/machine-type.md:18]: That's a really good point, I'll fix it. :+1:

**Pull request merged**